### PR TITLE
Add support to ignore diacritics

### PR DESCRIPTION
**Pull request by diogocorrea**
With this option, when given the a string like `cafe` is possible to highlight both `café` and `cafe`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jquery-highlight",
-  "version": "3.4.0",
+  "version": "3.5.0",
```
Comment (knownasilya) [inline package.json]: Could you remove this bump. I usually do this with my own process when releasing.

**Comment by diogocorrea** [comment]
I used the https://prettier.io/ to format the code and it caused some diff changes. Let me know if its ok for you.

**Comment by knownasilya** [comment]
@diogocorrea I'm 👍 on the prettier changes, but would like them separately so it's clear what this PR does. Hope that's alright.

**Comment by diogocorrea** [comment]
I just put them on a separate commit.

**Comment by knownasilya** [comment]
Released as v3.5.0

**Pull request merged**